spec/services/weather_service_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'spec_helper'
+
+describe 'weather service' do
+  subject(:weather_service) { WeatherService.new('sydney') }
+
+  it 'instantiates' do
+    expect(weather_service).not_to be_nil
+    expect(weather_service.city).to eq('sydney')
+  end
+
+  it '#get_city_code' do
+    expect(weather_service.get_city_code).to eq(1105779)
+    expect(WeatherService.new('melbourne').get_city_code).to eq(1103816)
+  end
+
+  it '#get_forecast_data' do
+    expect(weather_service.get_forecast_data).to be_an_instance_of(Array)
```

Comment:
If you were working on a flight, this spec would fail. Why? What could you do to remove this brittleness?